### PR TITLE
added nsfw channel support

### PR DESCRIPTION
**Pull request by jagrosh**

### CODE REVIEW

**Comment by DV8FromTheWorld** [comment]
Doesn't handle different letter casing.

File: src/main/java/net/dv8tion/jda/core/entities/impl/TextChannelImpl.java [hunk 1]
```diff
@@ -223,6 +223,11 @@ public String getTopic()
     }
 
     @Override
+    public boolean isNSFW() {
+        return name.equals("nsfw") || name.startsWith("nsfw-");
```
Comment (ShanerX) [inline src/main/java/net/dv8tion/jda/core/entities/impl/TextChannelImpl.java]: Change to ``return name.equalsIgnoreCase("nsfw") || name.toLowerCase().startsWith("nsfw-");`` should fix it

**Comment by MinnDevelopment** [comment]
Does the casing matter for text channel names?

**Comment by aurieh** [comment]
It doesn't.

**Comment by DV8FromTheWorld** [comment]
I really don't like this "feature", but it is now squashable.

**Pull request merged**